src/CONFIG.js

```diff
@@ -1,51 +1,53 @@
 import _ from 'underscore';
+import lodashGet from 'lodash.get';
 import {Platform} from 'react-native';
 import Config from 'react-native-config';
+import getPlatform from './libs/getPlatform/index';
+
+// Set default values to contributor friendly values to make develo:pment work out of the box without an .env file
+let expensifyCashURL = lodashGet(Config, 'URL_EXPENSIFY_CASH', 'https://expensify.cash/');
+let expensifyURL = lodashGet(Config, 'EXPENSIFY_URL_COM', 'https://www.expensify.com/');
+let ngrokURL = lodashGet(Config, 'NGROK_URL', null);
+const useNgrok = lodashGet(Config, 'USE_NGROK', 'false');
+const useWebProxy = lodashGet(Config, 'USE_WEB_PROXY', 'true');
```

Comment:
NAB, it would be a small code clarity boost if we made these two `Boolean` in this step so we don't have to do it later in the more complex conditionals. Which we can do by just adding an `=== 'true'` to the end of both of these.